src/chapters/booleans-and-conditionals/logical-operators.rst

```diff
@@ -0,0 +1,273 @@
+
+The logical NOT operator, ``!``, takes only a single operand and reverses its boolean value.
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+
+      console.log(! true);
+      console.log(! false);
+
+   **Output**
+
+   ::
+
+      false
+      true
+
+The operator ``!`` (sometimes called "bang") has the same semantic role as the word "not" in English. 
+
+.. admonition:: Example
+
+   .. sourcecode:: js
+
+      console.log( !(5 > 7) );
+      console.log( !('dog' === 'cat') );
+
+   **Output**
+
+   ::
+
+      true
+      true
+
+Both of the overall expressions in this example return ``true`` because without the ``!`` each is ``false``.
+
+Operator Precedence
+-------------------
+
+.. index::
+   single: operator; precedence
+
+We now have a number of operators in our toolkit. It is important to understand how these operators relate to each other with respect to **operator precedence**. Operator precedence is the set of rules that dictate in which order the operators are applied.
+
+JavaScript will always apply the logical NOT operator, ``!``, first. It then applies the arithmetic operators (``**`` is highest, then ``*`` and ``/``, then ``+`` and ``-``). Next comes the comparison operators. Finally, the logical AND and OR are applied last. 
+
+This means that the expression ``x * 5 >= 10 && y - 6 <= 20`` will be evaluated so as to first perform the arithmetic and then check the relationships. The ``&&`` evaluation will be done last. The order of evaluation is the same as if we were to use parentheses to group, as follows:
+
+.. sourcecode:: js
+
+   ((x * 5) >= 10) && ((y - 6) <= 20)
+
+While parentheses make expression more readable, they are not strictly necessary due to default operator precedence.
+
+The following table lists operators in order of precedence, from highest (applied first) to lowest (applied last). A complete table for the entire language can be found in the `MDN JavaScript Documentation <https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence#Table>`_.
+
+.. list-table:: Operator Precedence
+   :widths: auto
+   :header-rows: 1
+
+   * - Precedence
+     - Category
+     - Operators
+   * - (highest)
+     - Logical NOT
+     - ``!``
+   * -
+     - Exponentiation
+     - ``**``
+   * -
+     - Multiplication and division
+     - ``*``, ``/``, ``%``
+   * -
+     - Addition and subtraction
+     - ``+``, ``-``
+   * -
+     - Comparison
+     - ``<=``, ``>=``, ``>``, ``<``
+   * -
+     - Equality
+     - ``===``, ``!==``, ``==``, ``!=``
+   * -
+     - Logical AND
+     - ``&&``
+   * - (lowest)
+     - Logical OR
+     - ``||``
+
+Truth Tables
+------------
+
+.. index:: ! truth table
+
+**Truth tables** help us understand how logical operators work by calculating all of the possible return values of a boolean expression. Let's look at the truth table for ``&&``, which displays assumes we have two boolean expressions, A and B, joined by ``&&``.
```

Comment:
Typo: Remote 'displays'.  

"Let's look at the truth table for ``&&``, which assumes we have two boolean expressions, A and B, joined by ``&&``."